src/jvmMain/kotlin/com/epam/drill/agent/instrument/InstrumentationErrorHandler.kt

```diff
@@ -0,0 +1,33 @@
+package com.epam.drill.agent.instrument
+
+import com.alibaba.ttl.threadpool.agent.internal.javassist.*
+import com.epam.drill.logger.*
+
+inline fun CtMethod.insertCode(
```

Comment:
The "insertCode" function name does not say anything about what is happening in it, mb rename it ?

Reply:
rename: wrapException